frontends/web/src/routes/buy/pocket.tsx

```diff
@@ -152,6 +102,113 @@ export const Pocket = ({ code }: TProps) => {
     current.contentWindow?.postMessage(message, '*');
   };
 
+  const handleRequestAddress = (message: RequestAddressV0Message) => {
+    signing = true;
+    const addressType = message.withScriptType ? String(message.withScriptType) : '';
+    const withMessageSignature = message.withMessageSignature ? String(message.withMessageSignature) : '';
+    signAddress(
+      addressType,
+      withMessageSignature,
+      code)
+      .then(response => {
+        signing = false;
+        if (response.success) {
+          sendAddress(response.address, response.signature);
+        } else {
+          if (response.errorCode !== 'userAbort') {
+            alertUser(t('genericError'));
+            console.log('error: ' + response.errorMessage);
+          }
+        }
+      });
+
+  };
+
+  const handleVerifyAddress = (address: string) => {
+    setVerifying(true);
+    verifyAddress(address, code)
+      .then(response => {
+        setVerifying(false);
+        if (!response.success) {
+          if (response.errorCode === 'addressNotFound') {
+            // This should not happen, unless the user receives a tx on the same address between the message signing
+            // and the address verification.
+            alertUser(t('buy.pocket.usedAddress', { address:  address }));
+          } else {
+            alertUser(t('genericError'));
+            console.log('error: ' + response.errorMessage);
+          }
+        }
+      });
+  };
+
+  const sendXpub = () => {
+    if (accountInfo) {
+      let bitcoinSimple = accountInfo.signingConfigurations[0].bitcoinSimple;
```

Comment:
Const instead of let here makes sense:

```suggestion
      const bitcoinSimple = accountInfo.signingConfigurations[0].bitcoinSimple;
```

also there are some `var`s in this file. We can change those to `let`.

Reply:
Good points! Done :pray: